recipe/single_patch.patch

```diff
@@ -189,11 +189,14 @@ index 923da1b..d5df60b 100644
 +
 +        find_library(PLUTOSVG_LIBRARIES plutovg REQUIRED)
 +        target_link_libraries(imgui PRIVATE ${PLUTOSVG_LIBRARIES})
++        find_library(FREETYPE_LIBRARIES freetype REQUIRED)
++        target_link_libraries(imgui PRIVATE ${FREETYPE_LIBRARIES})
++        target_compile_definitions(imgui PUBLIC IMGUI_ENABLE_FREETYPE)
```

Comment:
@pthom this should hopefully address the issues with freetype.

Reply:
It is doing better, see this error message in the builds:

Can not link to _ZN13ImGuiFreeType21GetBuilderForFreeTypeEv 

This means that ImGui is now compiled with the correct option.
However, some additional modifications are required:

See https://github.com/pthom/hello_imgui/blob/1aab7451bc2f177d4d18a71743d03f4167bf4147/hello_imgui_cmake/hello_imgui_build_lib.cmake#L435-L443

```
    # 1. imgui must be linked with freetype, with the correct flag (I think this is done already)
    target_link_libraries(imgui PUBLIC ${HIM_FREETYPE_LINKED_LIBRARY})
    target_compile_definitions(imgui PUBLIC IMGUI_ENABLE_FREETYPE)

    # 2. Add required sources to imgui
    target_sources(imgui PRIVATE
        ${HELLOIMGUI_IMGUI_SOURCE_DIR}/misc/freetype/imgui_freetype.cpp
        ${HELLOIMGUI_IMGUI_SOURCE_DIR}/misc/freetype/imgui_freetype.h)

    # 2. Add support for wchar32 (for emojis, and other unicode characters) inside imgui
    target_compile_definitions(imgui PUBLIC IMGUI_USE_WCHAR32)
    ```

Reply:
Ah man. I definitely went down the rabbit hole finding this.
Glad I learned and that you confirmed things 

^_^